project/src/components/layout.tsx

```diff
@@ -0,0 +1,18 @@
+import { Outlet } from 'react-router-dom';
+import {ReactNode} from 'react';
+import Header from './header/header';
+
+type LayoutProps = {
+  className: string;
+  children: ReactNode;
+}
+
+const Layout = ({children, className}: LayoutProps): JSX.Element => (
+  <div className={className}>
```

Comment:
Здесь лучше оставить тот класс который у нас всегда есть -  page. Для того чтобы смержить классы, здесь подойдет и простая конкатенация, но в случае css модулей лучше использовать classnames или clsx
```suggestion
  <div className={`page ${className}`}>
```